docs/scos/dev/front-end-development/202212.0/oryx/oryx-compositions.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Oryx: Compositions"
+description: Compositions are used to organize components and provide layout in Oryx applications
+last_updated: June 8, 2023
+template: concept-topic-template
+---
+
+Compositions in Oryx are a powerful tool for organizing components and defining their layout. The removes page specific layout concerns from the component implementation, which makes the components less opinionated and therefor better reusable.
+
+## Overview
+
+A composition contains a list of components which are rendered in the DOM in their specific order. A composition itself is a component, which means that you can easily created nested composition structures. In fact, a page itself is a composition.
+
+Compositions in Oryx take a data-driven approach, allowing you to configure the composition using external data sources. This approach offers several potential benefits:
+
+- **Avoid Hard-Coded Page Layout**: By using data to configure compositions, you can avoid hard-coding the page layout in your application code. Instead, you can define the structure and layout of your pages using external data, making it more flexible and easier to customize.
```

Comment:
Maybe we can also mention that we are separating application/component code from the layout and as a result the customer will benefit from it during upgrades.